src/webgpu/api/operation/rendering/depth_clip_clamp.spec.ts

```diff
@@ -176,7 +172,7 @@ have unexpected values then get drawn to the color buffer, which is later checke
         topology: 'point-list',
         // `|| undefined` is a workaround for Chromium not allowing `false` here
         // when the feature is unavailable.
```

Comment:
And this workaround isn't needed anymore since when we update chromium we'll fix the bug.
```suggestion
```